src/vaults/apys/implementations/convex.utils.js

```diff
@@ -722,6 +722,16 @@ const pools = [
     name: 'mim',
     id: 40,
   },
+  {
+    lptoken: '0x19b080FE1ffA0553469D20Ca36219F17Fcf03859',
+    token: '0x864510e93c38c771adc1b67308ce0b7c4aa1aa9e',
+    gauge: '',
```

Comment:
gauge = 0x99fb76F75501039089AAC8f20f487bf84E51d76F

https://etherscan.io/address/0x99fb76f75501039089aac8f20f487bf84e51d76f

Reply:
how can I find out ```gauge``` for some tokens?

Reply:
Check "holders" for the Curve LP token on etherscan. 99.9% will be in the gauge